features/top-level-await.yml

```
@@ -0,0 +1,6 @@
+name: Top-level await
+description: Use the `await` keyword outside of async functions and at the top level of modules.
```

"outside of async functions and at the top level of modules" sounds a little bit (to me) like there might be other places where top-level await can be used.


```suggestion
description: The `await` keyword, when used at the top level of a module (outside of an async function), delays parent module execution until after a promise fulfills.
```